tests/impl.cpp

```diff
@@ -3901,7 +3901,16 @@ result_t test_mm_comilt_sd(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_comineq_sd(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const double *_a = (const double *) impl.mTestFloatPointer1;
+    const double *_b = (const double *) impl.mTestFloatPointer2;
+    int32_t _c = (_a[0] != _b[0]) ? 1 : 0;
```

Comment:
Shall we add the checking for `NaN` value?

Reply:
I won't add checking NaN, since I won't handle the `NaN` condition here.